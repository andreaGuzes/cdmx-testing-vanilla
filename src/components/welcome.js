// aqui exportaras las funciones que necesites

export function bienvenida(target){
  const html=  `<h1>Beauty Culture</h1>
          <div id="dates">
          <h2>Iniciar sesión</h2>
          <form class="form" id="form">
          <h4>Correo electrónico</h4>
          <input type="email" class="form-control" id="useremail" placeholder="Ej: usuario@servidor.com" required>
          <h4>Contraseña</h4>
          <input type="password" class="form-control" id="password" placeholder="Ingrese contraseña:" required>
          <button type="submit" class="btn btn-outline-success">Iniciar</button>
          </div>`

        target.innerHTML= html
};

// const form= document.getElementById('form')
// const useremail= document.getElementById('useremail')
// const password= document.getElementById('password')

// form.addEventListener('submit', function (event){
//  event.preventDefault();
// //  let users= Array[
// //     {
// //        correo: useremail.value,
// //        contraseña: password.value
// //     }
// //  ];
//  console.log('click');
// });
