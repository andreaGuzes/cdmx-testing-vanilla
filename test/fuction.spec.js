/* eslint-disable quotes */
import { bienvenida } from '../src/components/welcome.js';

const Dom = document.createElement('div');
describe('new suite', () => {
  test('see function dom', () => {
    const element = bienvenida(Dom);
    expect(element).toEqual(`<h1>Bienvenida a mi Red social</h1>`);
  });

  test('See Snapshot', () => {
    const element = bienvenida(Dom);
    expect(element).toMatchSnapshot();
  });
});
