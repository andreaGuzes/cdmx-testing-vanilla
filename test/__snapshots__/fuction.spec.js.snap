// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new suite See Snapshot 1`] = `"<h1>Bienvenida a mi Red social</h1>"`;
